tests/test_embedding_chain_breaks.py

```diff
@@ -135,7 +209,13 @@ def test_four_chains(self):
         self.assertEqual(set().union(*samples), {-1, 1})  # should be spin-valued
 
 
-class TestMinimizeEnergy(unittest.TestCase):
+class TestMinimizeEnergy(TestChainBreakResolutionAPI, unittest.TestCase):
+    # for the API tests
+    def setUp(self):
+        embedding = dict(zip('abcdefghijk', self.chains))
+        bqm = dimod.BinaryQuadraticModel.from_ising({v: 0 for v in embedding}, {})
+        self.chain_break_method = dwave.embedding.MinimizeEnergy(bqm, embedding)
```

Comment:
I would have expected `self.chain_break_method = dwave.embedding.MinimizeEnergy` and all the uses of `cbm` in this class to instead be `self.chain_break_method`.

Reply:
`MinimizeEnergy` is a callable object so it returns the chain break method that depends on the specific bqm/embedding. In the case below we are using a different embedding than the one in `setUp` which is why we don't use the class one.